js/reference.js

```diff
@@ -4014,6 +4014,87 @@
             return this._cascadingDeletedItems;
         },
 
+        /**
+         * If prefill object is defined and has the required attributes, will return
+         * a BulkCreateForeignKeyObject object with the necessary objects used for a association modal picker
+         *
+         * @type {ERMrest.BulkCreateForeignKeyObject}
+         */
+        get bulkCreateForeignKeyObject() {
+            if (this._bulkCreateForeignKeyObject === undefined) {
+                verify(false, 'Call "computeBulkCreateForeignKeyObject" with the prefill object first');
```

Comment:
I feel like instead of throwing error we should just do

```
this._bulkCreateForeignKeyObject = null;
```

So if the compute is called, it will return the proper value. And if it's not called, it will just return `null`. We then don't need to necessarily call compute in all the different places in chaise just to make sure this doesn't throw any errors.